### PR TITLE
Convert renderers to presenters

### PR DESCRIPTION
**Pull request by chrislo**
This PR moves the `SectionRenderer` and `ManualRenderer` classes into the `app/presenters` directory, renaming them as `SectionPresenter` and `ManaualPresenter`. This is more consistent with how presenter objects are used in, for example, [publisher](https://github.com/alphagov/publisher/tree/master/app/presenters) and [specialist publisher](https://github.com/alphagov/specialist-publisher/tree/master/app/presenters). 

I've started to refactor `ManualPresenter` too removing delegation to make it clear what the interface of the presenter is. There's quite a bit more to do on `SectionPresenter`, but I don't think this PR leaves things in a too inconsistent state and can be landed while I work on `SectionPresenter`. 

This is part of #995.

### CODE REVIEW

File: app/presenters/manual_presenter.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def summary
   end
 
   def body
-    GovspeakToHTMLRenderer.create.call(manual).attributes.fetch(:body)
+    GovspeakHtmlConverter.new.call(manual.body)
```
Comment (floehopper) [inline app/presenters/manual_presenter.rb]: I think this change makes the `require "govspeak_to_html_renderer"` statement at the top of the file redundant.

File: app/presenters/manual_presenter.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
 require "govspeak_to_html_renderer"
 
 class ManualPresenter
-  def call(manual)
+  attr_reader :manual
```
Comment (floehopper) [inline app/presenters/manual_presenter.rb]: Minor/subjective: Does this reader method need to be public or do we even need it at all?

**Comment by chrislo** [comment]
Force pushing before merging.

**Comment by chrislo** [comment]
Thanks @floehopper! I think it's a good idea to not have presented objects returned from the services. I'll create an issue to investigate that.

**Comment by chrislo** [comment]
I raised #1054 to capture the above.

**Pull request merged**